explore/src/main/scala/explore/HomeComponent.scala

```diff
@@ -40,28 +42,36 @@ object HomeComponent {
     ScalaComponent
       .builder[Unit]("Home")
       .initialState(0)
-      .renderPS { (_, _, i) =>
+      .renderPS { (_, _, _) =>
         <.div(
           ^.cls := "rgl-area",
           SizeMe() { s =>
-            <.div(
-              ResponsiveReactGridLayout(
-                s.width,
-                margin           = (5: JsNumber, 5: JsNumber),
-                containerPadding = (5: JsNumber, 5: JsNumber),
-                className        = "layout",
-                rowHeight        = 30,
-                draggableHandle  = ".tileTitle",
-                onLayoutChange   = (a, b) => Callback.log(a.toString) *> Callback.log(b.toString),
-                layouts          = layouts
-              )(
-                <.div(^.key := "tpe",
-                      ^.cls := "tile",
-                      Tile(Tile.Props("Conditions"), Conditions())),
-                <.div(^.key := "coords", ^.cls := "tile", Tile(Tile.Props("Coordinates"), Imag())),
-                <.div(^.key := "doc",    ^.cls := "tile", Tile(Tile.Props("Target Position"), Tpe(i)))
+            if(!js.isUndefined(s.width)) {
+              <.div(
+                ResponsiveReactGridLayout(
+                  s.width,
+                  margin           = (5: JsNumber, 5: JsNumber),
+                  containerPadding = (5: JsNumber, 5: JsNumber),
+                  className        = "layout",
+                  rowHeight        = 30,
+                  draggableHandle  = ".tileTitle",
+                  onLayoutChange   = (a, b) => Callback.log(a.toString) *> Callback.log(b.toString),
+                  layouts          = layouts
+                )(
+                  <.div(^.key := "tpe",
+                        ^.cls := "tile",
+                        Tile(Tile.Props("Conditions"), Conditions())),
+                  <.div(^.key := "coords", ^.cls := "tile", Tile(Tile.Props("Coordinates"), Imag())),                  
+                    <.div(^.key := "doc",    ^.cls := "tile", Tile(Tile.Props("Target Position"), 
+                      Views.target.flow{ targetOpt =>
```

Comment:
Calling `.flow` on a `View` creates a component. We should move this out of the render section and just use the component here.

Reply:
👍